knative-operator/pkg/common/util.go

```diff
@@ -69,11 +75,11 @@ func BuildImageOverrideMapFromEnviron(environ []string) map[string]string {
 }
 
 // SetOwnerAnnotations is a transformer to set owner annotations on given object
-func SetOwnerAnnotations(instance *operatorv1alpha1.KnativeServing) mf.Transformer {
+func SetOwnerAnnotations(meta metav1.ObjectMeta, ownerNameAnnotationKey, ownerNamespaceAnnotationKey string) mf.Transformer {
```

Comment:
I see why you did the ObjectMeta thing here, but I think it'd be somewhat clearer if we just passed name, namespace as a string here.

Reply:
sgtm.

let me update!

Reply:
Actually thinking about it: We might as well pass a `map[string]string` in to make the mapping VERY clear :joy:. We don't win anything by passing it separately anyway.

Reply:
As such, we might as well rename the function to `SetAnnotations`

Reply:
Pushed another commit now that does that.
However, I changed the logic slightly:
- The old function existed in the repo since beginning was clearing the existing annotations and setting our own stuff
- New function keeps the existing ones, adds new ones and if there's already with the same key, it overwrites that